session_id.go

```diff
@@ -2,13 +2,18 @@ package quickfix
 
 import (
 	"fmt"
+	"github.com/quickfixgo/quickfix/enum"
 )
 
 // SessionID is a unique identifer of a Session
 type SessionID struct {
 	BeginString, TargetCompID, TargetSubID, TargetLocationID, SenderCompID, SenderSubID, SenderLocationID, Qualifier string
 }
 
+func (s SessionID) IsFIXT() bool {
```

Comment:
Kind of ashamed to request this, but in the interests of not upsetting golint anymore, can you add a `//IsFIXT...` comment for this?